### PR TITLE
Add "swish" activation layer

### PR DESCRIPTION
**Pull request by yingriyanlong**

### CODE REVIEW

**Comment by Dobiasd** [comment]
Hi! :slightly_smiling_face:

This is a very nice contribution, thanks a lot! :+1:

Could you please also add a "swish" to [this block of test cases](https://github.com/Dobiasd/frugally-deep/blob/5ac6cfeb4b7c778cd625e0ded07a1a49e76cae4d/keras_export/generate_test_models.py#L344-L351)? If the CI of this pull request then succeeds, we know it's working as intended.

File: include/fdeep/layers/swish_layer.hpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// Distributed under the MIT License.
+// (See accompanying LICENSE file or at
+//  https://opensource.org/licenses/MIT)
+
+#pragma once
+
+#include "fdeep/layers/activation_layer.hpp"
+#include "fdeep/recurrent_ops.hpp"
+
+#include <limits>
```
Comment (Dobiasd) [inline include/fdeep/layers/swish_layer.hpp:12]: I guess this `#include` is not needed. :)

**Comment by yingriyanlong** [comment]
> Hi! 🙂
> 
> This is a very nice contribution, thanks a lot! 👍
> 
> Could you please also add a "swish" to [this block of test cases](https://github.com/Dobiasd/frugally-deep/blob/5ac6cfeb4b7c778cd625e0ded07a1a49e76cae4d/keras_export/generate_test_models.py#L344-L351)? If the CI of this pull request then succeeds, we know it's working as intended.

Hi!
I have made the changes and done local testing.

**Comment by Dobiasd** [comment]
Looks good! :+1:
Thanks. :heart:

**Pull request merged**